Doc Update with the missing steps in INSTALL_GUIDE.md

Ishtiyaque-Alam opened the pull request:
Updated the missing steps in INSTALL_GUIDE.md to avoid any Setup error

<!-- What issue does this PR close? -->
Closes #


## 📝 Description
The INSTALL_GUIDE.md was missing some steps to setup the Supabase Table, proper guide to the flask app path 
which raised errors  such as 'No Flask app-Found. '

<!-- A clear and concise description of what this pull request does. -->
<!-- Include any context or background information if necessary. -->

## 🔧 Changes Made
Added the step to setup the Supabase table.
Added the steps required to reach and run the Flask App


<!-- List the changes you made in this pull request. -->
<!-- For example:
- Fixed a bug in the login flow.
- Added a new feature for user profile customization.
- Updated documentation for API endpoints.
-->

## 📷 Screenshots or Visual Changes (if applicable)

<!-- Attach screenshots or GIFs to show visual changes, if any. -->
<!-- Drag and drop screenshots here or provide a description of visual updates. -->

## 🤝 Collaboration
NA
<!-- If you collaborated with someone on this pull request, mention their GitHub username or name here. -->
Collaborated with: `@username` (optional)


### ✅ Checklist

- [x] I have read the contributing guidelines.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation (if applicable).
- [x] Any dependent changes have been merged and published in downstream modules.



<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Documentation**
  * Reorganized installation guide with renumbered startup steps and an optional Docker Desktop startup flow
  * Clarified cross-platform backend and frontend startup commands (now noting npm run dev)
  * Added a Supabase table creation section with SQL paste steps and a warning about potential "Table Not Found" errors
  * Introduced FalkorDB CodeGraph and Supabase descriptions, reordered database context, and applied minor environment/setup wording refinements

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Ishtiyaque-Alam committed:
Doc Update with the missing steps

Updated the missing steps in INSTALL_GUIDE.md to avoid any Setup error

Ishtiyaque-Alam committed:
Update installation guide section numbering

smokeyScraper commented:
Could you please review the coderabbit-suggested changes once @Ishtiyaque-Alam ?
Thanks!

Ishtiyaque-Alam committed:
Merge branch 'AOSSIE-Org:main' into Ishtiyaque-Alam-Docs-Updating-the-Install.md-and-Readme.md-with-missing-steps

Ishtiyaque-Alam committed:
Update installation guide for clarity and path adjustments

Clarified installation steps by specifying the directory context for creating integration tables and adjusting the path for additional tables.

Ishtiyaque-Alam committed:
Revise Docker setup instructions in INSTALL_GUIDE.md

Updated installation guide to clarify Docker setup steps and corrected formatting.

Ishtiyaque-Alam commented:
@smokeyScraper I have made the suggested changes , please review it .

Ishtiyaque-Alam committed:
Minor change to repo location

Ishtiyaque-Alam commented on docs/INSTALL_GUIDE.md:
````diff
@@ -54,27 +54,43 @@ cp env.example .env
 nano .env  # or use your preferred editor
 ```
+6. **Create the required Supabase Tables**
+```sh
+# This can be done using the Supabase CLI or by pasting the SQL scripts directly into the SQL editor in Supabase.
+# If you skip this step, you'll likely get a 'Table Not Found' error.
+
+# Step 1: Create integration tables (from repo root)
+cd backend/database
+# Open the file 01_create_integration_tables.sql, copy all its contents, 
+# and paste it into the SQL editor of your Supabase project.
 
-6. **Set up Docker container**
+# Step 2: Create additional tables
+cd ../../backend/app/database/supabase/scripts
+# Copy and paste the SQL scripts in this folder into the SQL editor as well.
+```
````
I have made the desired changes
